examples/light-switch-app/silabs/include/BindingHandler.h

```diff
@@ -19,15 +19,57 @@
 #include "app-common/zap-generated/ids/Clusters.h"
 #include "app-common/zap-generated/ids/Commands.h"
 #include "lib/core/CHIPError.h"
+#include <platform/CHIPDeviceLayer.h>
+#include <app/clusters/bindings/bindings.h>
+#include <variant>
+
+using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters::LevelControl;
 
 CHIP_ERROR InitBindingHandler();
 void SwitchWorkerFunction(intptr_t context);
 void BindingWorkerFunction(intptr_t context);
 
+
 struct BindingCommandData
 {
     chip::EndpointId localEndpointId = 1;
     chip::CommandId commandId;
     chip::ClusterId clusterId;
     bool isGroup = false;
+
```

Comment:
To reduce some of the duplication, you can create a Command class with the optionsMasks and optionsOverride members and have all the others structucres inherits from it.

Reply:
made this change